tests/estimators/classification/test_input_filter.py

```diff
@@ -57,304 +57,304 @@ def test_fit(self):
-                -0.00239714,
-                -0.00122619,
-                0.0,
-                0.0,
-                -0.00520899,
-                -0.00046105,
-                0.00414874,
-                -0.00171095,
-                0.00429184,
-                0.0075138,
-                0.00792442,
-                0.0019566,
-                0.00035517,
-                0.00504575,
-                -0.00037397,
-                0.00022343,
-                -0.00530034,
-                0.0020528,
-                0.0,
-                0.0,
-                0.0,
-                0.0,
-                0.0,
-            ]
-        )
-        np.testing.assert_array_almost_equal(gradients[0, 0, :, 14, 0], expected_gradients_2, decimal=4)
-
-    def test_loss_gradient(self):
-        classifier = get_image_classifier_kr_tf()
-
-        # Test gradient
-        gradients = classifier.loss_gradient(x=self.x_test_mnist, y=self.y_test_mnist)
-
-        self.assertTrue(gradients.shape == (NB_TEST, 28, 28, 1))
-
-        expected_gradients_1 = np.asarray(
-            [
-                0.0559206,
-                0.05338925,
-                0.0648919,
-                0.07925165,
-                -0.04029291,
-                -0.11281465,
-                0.01850601,
-                0.00325054,
-                0.08163195,
-                0.03333949,
-                0.031766,
-                -0.02420463,
-                -0.07815556,
-                -0.04698735,
-                0.10711591,
-                0.04086434,
-                -0.03441073,
-                0.01071284,
-                -0.04229195,
-                -0.01386157,
-                0.02827487,
-                0.0,
-                0.0,
-                0.0,
-                0.0,
-                0.0,
-                0.0,
-                0.0,
-            ]
-        )
-        np.testing.assert_array_almost_equal(gradients[0, 14, :, 0], expected_gradients_1, decimal=4)
-
-        expected_gradients_2 = np.asarray(
-            [
-                0.00210803,
-                0.00213919,
-                0.00520981,
-                0.00548001,
-                -0.0023059,
-                0.00432077,
-                0.00274945,
-                0.0,
-                0.0,
-                -0.0583441,
-                -0.00616604,
-                0.0526219,
-                -0.02373985,
-                0.05273106,
-                0.10711591,
-                0.12773865,
-                0.0689289,
-                0.01337799,
-                0.10032021,
-                0.01681096,
-                -0.00028647,
-                -0.05588859,
-                0.01474165,
-                0.0,
-                0.0,
-                0.0,
-                0.0,
-                0.0,
-            ]
-        )
-        np.testing.assert_array_almost_equal(gradients[0, :, 14, 0], expected_gradients_2, decimal=4)
+    # def test_class_gradient(self):
```

Comment:
Why do you comment out the test, should we remove it if it is not necessary?

Reply:
I would like to keep it because we plan to reintroduce the feature later.